packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -0,0 +1,147 @@
+    supportText?: React.ReactNode;
+    endAdornment?: React.ReactNode;
+    collapsible?: boolean;
+    initialCollapsed?: boolean;
+    disablePadding?: boolean;
+    componentsProps?: FieldSetComponentsProps;
+}
+
+export type FieldSetClassKey = "header" | "headerColumn" | "title" | "supportText" | "endAdornment" | "placeholder" | "children" | "disablePadding";
+
+const styles = (theme: Theme) =>
+    createStyles<FieldSetClassKey, FieldSetProps>({
+        header: {
+            display: "flex",
+            flexDirection: "row-reverse",
+            padding: "0 10px",
+            height: "80px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "0 20px",
+            },
+        },
+        headerColumn: {
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "flex-start",
+            alignContent: "center",
+            padding: "0px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "10px",
+            },
+        },
+        title: {
+            display: "flex",
+            alignItems: "center",
+            fontWeight: theme.typography.fontWeightMedium,
+            fontSize: "14pt",
+            textTransform: "uppercase",
+            color: theme.palette.text.primary,
+        },
+        supportText: {
+            fontSize: "10pt",
+            lineHeight: "18px",
+            color: theme.palette.text.secondary,
+        },
+        endAdornment: { display: "flex", alignItems: "center" },
+        placeholder: {
+            flexGrow: 1,
+            boxSizing: "inherit",
+            userSelect: "none",
+        },
+        children: {
+            display: "flex",
+            flexDirection: "column",
+            borderTop: `solid ${theme.palette.divider}`,
+            padding: "20px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "40px",
+            },
+            "&$disablePadding": {
+                padding: "0px",
+            },
+        },
+        disablePadding: {},
+    });
+
+function FieldSet({
+    title,
+    supportText,
+    endAdornment,
+    children,
+    collapsible = true,
+    initialCollapsed = false,
+    disablePadding = false,
+    componentsProps,
+    classes,
+}: FieldSetProps & WithStyles<typeof styles>): React.ReactElement {
+    const [expanded, setExpanded] = React.useState(!initialCollapsed);
+
+    const handleChange = (event: React.SyntheticEvent, isExpanded: boolean) => {
+        setExpanded(isExpanded);
+    };
+
+    return (
+        <MuiAccordion
+            expanded={expanded}
+            onChange={
+                collapsible
+                    ? handleChange
+                    : () => {
+                          /* do nothing */
+                      }
+            }
+        >
+            <MuiAccordionSummary
+                classes={{ root: classes.header }}
+                expandIcon={collapsible && <ArrowForwardIosSharpIcon />}
```

Comment:
Does is remove the expand icon when the field set is not collapsible?

Reply:
yes
<img width="594" alt="Screenshot 2023-10-13 at 10 31 05" src="https://github.com/vivid-planet/comet/assets/72875628/6c4adae7-8d49-433e-9ecb-d3ddfb4424b0">